pandas/core/frame.py

```diff
@@ -1638,6 +1638,11 @@ def to_latex(self, buf=None, columns=None, col_space=None, header=True,
             Default encoding is ascii in Python 2 and utf-8 in Python 3
         decimal : string, default '.'
             Character recognized as decimal separator, e.g. ',' in Europe
+        multicol : boolean, default False
+            Use \\multicol to enhance MultiIndex columns.
```

Comment:
I see that you're following what's already in the docstring, but don't we want the backslash to show up in the rendered docs? I.e. the rendered version of this should be `Use \multicol to enhance MultiIndex columns.` To do that, do we need to write a single `\` in the source, and prefix the docstring with an `r`?